app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str | None:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("You have to be vaccinated.")
+        elif visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("Your vaccine is expired.")
+        elif not visitor["wearing_a_mask"]:
+            raise NotWearingMaskError("You have to be in mask.")
+        return f"Welcome to {self.name}"
```

Comment:
Please use the get() method to verify the presence of a key and to fetch its value, as this approach can prevent KeyError exceptions .